src/app.js

```diff
@@ -260,32 +257,64 @@ class SAFEApp extends EventEmitter {
         let fileSize;
         let lengthToRead = consts.pubConsts.NFS_FILE_END;
         let endByte;
+        let data;
+        let multipart;
+        let rangeIsArray;
+        let mimeType;
 
-        // TODO: how do we handle multipart Reqs
-        if (options && options.range && typeof options.range.start !== 'undefined') {
-          range = options.range;
-          start = range.start;
+        if (options && options.range) {
+          rangeIsArray = Array.isArray(options.range);
           fileSize = await openedFile.size();
-          end = range.end || fileSize - 1;
-
+          range = options.range;
+          multipart = range.length > 1;
+          start = options.range.start || consts.pubConsts.NFS_FILE_START;
```

Comment:
So are we making the `start` optional as well? if so  we'd need to add tests for that